src/index.html

```diff
@@ -48,6 +48,16 @@ <h2 class="title fw-bold mb-3">Add your score</h2>
           </button>
         </form>
       </div>
+      <footer class="border-top fixed-bottom p-3">
```

Comment:
### Optional
- [ ] It would be better to sort the score based on the highest to the lowest so it looks like the real "Leaderboard"